src/test/testarb.annotated

```diff
@@ -13,62 +14,63 @@
         "description": "Welcome message on the Home screen",
         "placeholders": {
             "firstName": {}
-             ^^^^^^^^^[decoration]placeholder
         }
     },
-    "pageHomeInboxCount": "{count, plural, zero{I have {vehicle;;Type, select, sedn{Sedan} cabrolet{Solid roof cabriolet} tuck{16 wheel truck} oter{Other}} no new messages} one{You have 1 new message} other{You have {count} new messages}}",
+    "pageHomeInboxCount": "{count, plural, zero{I have {vehicle;;Type, select, sedn{Sedan} cabrolet{Solid roof cabriolet} tuck{16 wheel truck} other{Other}} no new messages} one{You have 1 new {counts} message} other{You have {count} new messages}}",
                             ^^^^^[decoration]placeholder
-                                                                                                                                                                                                                         ^^^^^[decoration]placeholder
+                            ^^^^^[decoration]placeholder
+                                                        ^^^^^^^^^^^^^[decoration]placeholder
+                                                        ^^^^^^^^^^^^^[decoration]placeholder
+                                                                                                                                                                                                  ^^^^^^[decoration]placeholder
+                                                                                                                                                                                                                                   ^^^^^[decoration]placeholder
                                                                               ^^^^^[decoration]select
-                                                                                           ^^^^^^^^[decoration]select
-                                                                                                                          ^^^^[decoration]select
-                                                                                                                                               ^^^^[decoration]select
+                                                                                          ^^^^^^^^^[decoration]select
+                                                                                                                         ^^^^^[decoration]select
```

Comment:
It probably doesn't matter for the colouring, but I think the leading spaces are now included in these ranges. If they end up being used for anything else (like diagnostics, or in error messages with quotes around the name) this might look a little odd.